tests/test_jsonrpc.py

```diff
@@ -125,8 +132,8 @@ def test_request_id_validation_error(echo, json_request):
             'code': -32600,
             'message': 'Invalid Request',
             'data': {'errors': [
-                {'loc': ['id'], 'msg': 'str type expected', 'type': 'type_error.str'},
-                {'loc': ['id'], 'msg': 'value is not a valid integer', 'type': 'type_error.integer'},
+                {'input': [123], 'loc': ['id', 'str'], 'msg': 'Input should be a valid string', 'type': 'string_type'},
+                {'input': [123], 'loc': ['id', 'int'], 'msg': 'Input should be a valid integer', 'type': 'int_type'}
```

Comment:
it's new format for errors

loc now mean field name and concrete annotation:

for declaration `id: Union[StrictStr, int, float, decimal.Decimal]` it will be

```
[
{'type': 'string_type', 'loc': ('id', 'str'), 'msg': 'Input should be a valid string', 'input': [123]},
{'type': 'int_type', 'loc': ('id', 'int'), 'msg': 'Input should be a valid integer', 'input': [123]},
{'type': 'float_type', 'loc': ('id', 'float'), 'msg': 'Input should be a valid number', 'input': [123]},
{'type': 'decimal_type', 'loc': ('id', 'decimal'), 'msg': 'Decimal input should be an integer, float, string or Decimal object', 'input': [123]}
]
```

Reply:
i resolve it, now loc pointing to field location in json and type (annotation) info cutted